Nautilus/Documentation/guides/assetbundles.md

```diff
@@ -0,0 +1,114 @@
+# Asset Bundles Guide
+
+This guide will show you how to take a 3d model file from a program such as Blender, and turn it into a GameObject which can then be used in your mod.
+
+Basic modding knowledge is required for this tutorial.
+Here are some things you will need to know:
+- Basic modding and C# knowledge
+- Basic knowledge of how to navigate the Unity game engine interface
+
+The first thing you will need to do (Assuming you have a model already) is download the Unity Hub.  
+This is where you will get the Unity version you need to use to export your asset bundle.
+
+The version Subnautica uses is 2019.4.36 for SN1, as well as Below Zero.
+If you can't find that exact version, just get the one closest to it.
+
+Here are the download links you will need:  
+- Unity hub: [https://unity.com/download](https://unity.com/download)
+- Unity version archive: [https://unity.com/releases/editor/archive](https://unity.com/releases/editor/archive)
+
+Once you've installed both, open Unity hub and click "New Project" from the top right.  
+Once there, select 3D Core, choose your project name and then click "Create Project".  
+![assetbundles-project-setup](../images/guides/assetbundles-project-setup.png)
+
+> [!NOTE]
+> If you get the error "<font color="red">Failed to resolve project template</font>", follow these steps:
+> 1. Locate where your editor is. 
+> You can do that by opening Unity Hub, going to Installs, and then right clicking on the version in question and clicking "Show in Explorer"
+> 2. Go to ``Editor\Data\Resources\PackageManager\ProjectTemplates``. You will find a ``manifest.json`` and ``UnityLicense.json`` files. delete them both.
+> 3. Kill Unity Hub in your task manager and open it again.
+> 
+> You should now be able to create your project!
+
+The Unity editor will now open. This may take a while.  
+Once the editor is open, drag your model from your file manager into the Scene window.
+
+You can either click on your model in the scene window or in the hierarchy on the left to select it.
+
+Next, click on ``Window`` at the top of the Unity editor, and then click ``Package Manager``.  
+Click on the dropdown at the top left and select ``Unity Registry`` if not on it already.  
+Search for "Asset Bundle Browser" and install it.
```

Comment:
Imo installing the asset bundle browser should be closer to the top, before importing the model.